### DISCUSSION TITLE
Anyone can make an example about Role Based Access Control from scratch

### DISCUSSION THREAD

**Discussion by Lehuyknight** [adonisjs/core#3729] [Help] [answered]
Hello, I got some problems with RBAC using Adonis Bouncer, anyone have simple example? I have some models here
[Permission.txt](https://github.com/adonisjs/core/files/8707332/Permission.txt)
[Role.txt](https://github.com/adonisjs/core/files/8707333/Role.txt)
[User.txt](https://github.com/adonisjs/core/files/8707334/User.txt)

**Comment by galaczi** [comment, marked as answer]
Check out Tom Gobich's repo here:

https://github.com/jagr-co/jagr.co/blob/main/app/Middleware/Role.ts

He also has an awesome Youtube series about Adonis.
